install-offline/set-up-namespaces.hbs.md

```diff
@@ -6,4 +6,5 @@
 
 ## <a id='next-steps'></a>Next steps
 
-- [Deploy an air-gapped workload](../getting-started/air-gap-workload.hbs.md)
+- [Work with Git repositories in air-gapped environments with Namespace Provisioner](../namespace-provisioner/use-case7.hbs.md) for the Namespace Provisioner mode.
```

Comment:
Do we need the namespace provisioner mode and legacy mode? The existing way of doing it might not be legacy

Reply:
Changed legacy to manual